Woodall_Robert/Assignments/names.py

```diff
@@ -0,0 +1,86 @@
+'''
+
+Part 1:
+Given the following list:
+
+students = [ 
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'},
+     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+]
+
+Create a program that outputs:
+
+Michael Jordan
+John Rosales
+Mark Guillen
+KB Tonel
+
+Part 2:
+Given the following dictionary:
+
+users = {
+ 'Students': [ 
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'},
+     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+  ],
+ 'Instructors': [
+     {'first_name' : 'Michael', 'last_name' : 'Choi'},
+     {'first_name' : 'Martin', 'last_name' : 'Puryear'}
+  ]
+ }
+
+Create a program that prints  the following format (including number of characters in each combined name):
+
+Students
+1 - MICHAEL JORDAN - 13
+2 - JOHN ROSALES - 11
+3 - MARK GUILLEN - 11
+4 - KB TONEL - 7
+Instructors
+1 - MICHAEL CHOI - 11
+2 - MARTIN PURYEAR - 13
+
+'''
+
+students = [ 
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'},
+     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+]
+
+# print just names for part 1
+for dict in students:
+    print('{} {}'.format(dict['first_name'], dict['last_name']))
+
+print # blank line
+
+users = {
+ 'Students': [ 
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'},
+     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+  ],
+ 'Instructors': [
+     {'first_name' : 'Michael', 'last_name' : 'Choi'},
+     {'first_name' : 'Martin', 'last_name' : 'Puryear'}
+  ]
+ }
+
+# print names with associated index and number of chars
+for user_item, user_list in users.iteritems():
+    dict_count = 0
+    print(user_item)
+
+    for dict in user_list:
```

Comment:
Good job.

In the future, don't use the term 'dict' as a variable, since that's also the name of a datatype. Better to use dct, or some other variable name to avoid confusion of other peopl reading your code.